plugins/jenkins-backend/README.md

```diff
@@ -8,6 +8,20 @@ This is the backend half of the 2 Jenkins plugins and is responsible for:
 - finding the appropriate job(s) on that instance for an entity
 - connecting to Jenkins and gathering data to present to the frontend
 
+## New Backend System
+
+The jenkins backend plugin has support for the [new backend system](https://backstage.io/docs/backend-system/), here's how you can set that up:
+
+In your `packages/backend/src/index.ts` make the following changes:
+
+```diff
+  import { createBackend } from '@backstage/backend-defaults';
+  const backend = createBackend();
+  // ... other feature additions
+backend.add(import('@backstage/plugin-jenkins-backend'));
+  backend.start();
```

Comment:
```suggestion
   import { createBackend } from '@backstage/backend-defaults';
   const backend = createBackend();
   // ... other feature additions
+  backend.add(import('@backstage/plugin-jenkins-backend'));
   backend.start();
```

Tweaking this a bit for the diff format

Reply:
Ah, I don't have permission to apply this myself. Will need you to apply this before merge